src/index.html

```diff
@@ -16,17 +16,38 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="container">
+      <div
+        class="block-big block-big--hover block-big--top block-big__form--focus"
+      >
+        <img
+          class="block-big-picture"
+          src="/src/images/Search.svg"
+          alt="DSF"
+        />
+        <input
+          class="block-big__form block-big__form--focus"
+          id="data-qa=«big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
+      <div class="block-small block-small--top">
+        <img
+          class="block-small-picture"
+          src="/src/images/Search.svg"
+          alt="picture"
+        />
+        <input
+          class="block-small__form block-small__form--focus"
+          id="data-qa=”small»"
```

Comment:
The `id` attribute is incorrectly formatted. It should be `id="data-qa-small"` instead of `id="data-qa=”small»"`.